digdag-standards/src/test/java/io/digdag/standards/operator/HttpOperatorFactoryTest.java

```diff
@@ -0,0 +1,77 @@
+package io.digdag.standards.operator;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import io.digdag.spi.Operator;
+import io.digdag.spi.TaskExecutionException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static io.digdag.core.workflow.OperatorTestingUtils.newContext;
+import static io.digdag.core.workflow.OperatorTestingUtils.newOperatorFactory;
+import static io.digdag.core.workflow.OperatorTestingUtils.newTaskRequest;
+import static io.digdag.core.workflow.WorkflowTestingUtils.loadYamlResource;
+import static org.junit.Assert.fail;
+
+public class HttpOperatorFactoryTest
+{
+	@Rule
+	public TemporaryFolder folder = new TemporaryFolder();
+
+	@Rule
+	public WireMockRule wireMockRule = new WireMockRule(8089);
```

Comment:
If another process is using this port number, this test fails, right? How about using a random port number instead? Maybe passing 0 would work.